### PR TITLE
Refactor: log only intercepted HTTP responses, not requests

### PR DESCRIPTION
**Pull request by daabr**
This merges two log lines into one.

The diff looks confusing for no reason, the best way to review this PR is to compare the entire function before and after this PR:

Before: https://github.com/autokitteh/autokitteh/blob/main/internal/backend/httpsvc/intercept.go#L62-L106

After: `https://github.com/autokitteh/autokitteh/blob/5d3d36b40043f38d737d3ef970bfd3e29e6b1018/internal/backend/httpsvc/intercept.go#L63-L101`

### CODE REVIEW

File: internal/backend/httpsvc/intercept.go [hunk 1]
```diff
@@ -15,14 +15,15 @@ import (
 
 type ctxKey string
 
-var t0CtxKey = ctxKey("t0")
+var startTimeCtxKey = ctxKey("t0")
```
Comment (efiShtain) [inline internal/backend/httpsvc/intercept.go:18]: Thanks for the meaningful name :)

File: internal/backend/httpsvc/intercept.go [hunk 2]
```diff
@@ -66,42 +67,36 @@ func intercept(z *zap.Logger, cfg *LoggerConfig, extractors []RequestLogExtracto
...
+		startTime := time.Now()
+		r = r.WithContext(context.WithValue(r.Context(), startTimeCtxKey, startTime))
 
 		next.ServeHTTP(rwi, r)
-		d := time.Since(t0)
+		d := time.Since(startTime)
 
 		w.Header().Add("X-AutoKitteh-Duration", d.Truncate(time.Microsecond).String())
 
+		level := cfg.ImportantLevel.Level()
```
Comment (efiShtain) [inline internal/backend/httpsvc/intercept.go:83]: Not sure what are you trying to do here

Could you elaborate?

Comment (daabr) [reply internal/backend/httpsvc/intercept.go:83]: The diff looks confusing for no reason, the best way to review this PR is to compare the entire function before and after - and then you see that I basically cancelled the first log entry, and moved its fields to the second log entry.

Comment (daabr) [reply internal/backend/httpsvc/intercept.go:83]: I updated the PR description with relevant links

File: internal/backend/httpsvc/intercept.go (hunk 1 above)
Comment (itayd) [reply internal/backend/httpsvc/intercept.go:18]: you people suck

**Review by itayd — APPROVED** [review]

**Pull request merged**